<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="17256" started-at="2018-03-05T18:43:46Z" finished-at="2018-03-05T18:44:04Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.automation.Sample.CucumberFramework.runner.TestCaseRunner@589838eb]" name="feature" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Default test" duration-ms="17256" started-at="2018-03-05T18:43:46Z" finished-at="2018-03-05T18:44:04Z">
      <class name="com.automation.Sample.CucumberFramework.runner.TestCaseRunner">
        <test-method status="PASS" signature="setUpClass()[pri:0, instance:com.automation.Sample.CucumberFramework.runner.TestCaseRunner@589838eb]" name="setUpClass" is-config="true" duration-ms="197" started-at="2018-03-06T00:13:46Z" finished-at="2018-03-06T00:13:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method status="FAIL" signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:com.automation.Sample.CucumberFramework.runner.TestCaseRunner@589838eb]" name="feature" duration-ms="16744" started-at="2018-03-06T00:13:47Z" description="Runs Cucumber Feature" data-provider="features" finished-at="2018-03-06T00:14:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Login To Facebook]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.AssertionError: Failing to check few things expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: Failing to check few things expected [true] but found [false]at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: java.lang.AssertionError: Failing to check few things expected [true] but found [false]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertTrue(Assert.java:44)at com.automation.Sample.CucumberFramework.pageobject.FacebookHomePage.verifyProfile(FacebookHomePage.java:35)at com.automation.Sample.CucumberFramework.stepdefinition.LoginToFacebook.user_is_able_to_successfull_login(LoginToFacebook.java:39)at âœ½.Then User is able to successfull login(featurefile/FacebookFirstPage/FacebookLogin.feature:10)]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method status="PASS" signature="writeExtentReport()[pri:0, instance:com.automation.Sample.CucumberFramework.runner.TestCaseRunner@589838eb]" name="writeExtentReport" is-config="true" duration-ms="17" started-at="2018-03-06T00:14:03Z" finished-at="2018-03-06T00:14:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- writeExtentReport -->
        <test-method status="PASS" signature="tearDownClass()[pri:0, instance:com.automation.Sample.CucumberFramework.runner.TestCaseRunner@589838eb]" name="tearDownClass" is-config="true" duration-ms="224" started-at="2018-03-06T00:14:03Z" finished-at="2018-03-06T00:14:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- com.automation.Sample.CucumberFramework.runner.TestCaseRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
